firebase-app-distribution/src/main/java/com/google/firebase/app/distribution/FirebaseAppDistribution.java

```diff
@@ -129,10 +133,12 @@ public UpdateTask updateIfNewReleaseAvailable() {
       if (cachedUpdateIfNewReleaseTask != null && !cachedUpdateIfNewReleaseTask.isComplete()) {
         return cachedUpdateIfNewReleaseTask;
       }
-
       cachedUpdateIfNewReleaseTask = new UpdateTaskImpl();
     }
-    checkForNewRelease()
+
+    showSignInDialog()
+        .onSuccessTask(unused -> signInTester())
```

Comment:
Just to be clear: even though checkForNewRelease() calls signInTester(), you're calling it here too just to be explicit?